content/en/dora_metrics/data_collected/_index.md

```diff
@@ -63,7 +63,14 @@ Datadog breaks down change lead time into the following metrics, which represent
 
 These metrics are only computed when the source of the repository metadata is GitHub, and there must be a pull request (PR) associated with a commit, if any. A commit is associated with a PR if the commit is first introduced to the target branch when merging that PR. If a commit does not have an associated PR, only `dora.time_to_deploy` and `dora.deploy_time` metrics are available.
 
-**Note:** These metrics are emitted for every commit and not per deployment.
+**Notes:**
+
+- These metrics are emitted for every commit and not per deployment.
+- Commits that are not meaningful changes (for example merge commits) are ignored.
+- If commits on a feature branch are rebased, the commits considered for the change lead time are the new commits created with the rebase and not the original commits.
```

Comment:
```suggestion
- For rebased commits on a feature branch, *change lead time* calculations consider the new commits created during the rebase, not the original commits.
```

Reply:
I wanted to double check this change: `rebased` here is used as a verb, ie: "when you rebase a commit" and it feels like the change switches it to an adjective. 

But I think the usage of commit is wrong there, we don't rebase commits, we rebase branches (my bad!). What do you think about this?

```suggestion
- For rebased feature branches, *change lead time* calculations consider the new commits created during the rebase, not the original commits.
- ```

Reply:
Ah! All good. Yeah, I agree with your proposed edits. 🚀